### PR TITLE
Close #13748: Refactor About window to class

### LINKED ISSUE #13748: Refactor window to class: About
See #13737 for more details.

### PR DESCRIPTION
**Pull request by vkhashimoto**
Closes #13748

### CODE REVIEW

**Review by tupaschoal — COMMENTED** [review]
As a general tip, when you're closing/fixing open issues, do add on your commit title or description things like "Closes #XXXXX" or "Fix #XXXXX". This will make the PR appear at the issue, showing people there you have a potential solution, and it will automatically close the issue once your PR is merged.

File: src/openrct2-ui/windows/About.cpp [hunk 1]
```diff
@@ -73,237 +74,207 @@ static rct_widget window_about_openrct2_widgets[] = {
...
-            window_close(w);
-            break;
-        case WIDX_TAB_ABOUT_OPENRCT2:
-        case WIDX_TAB_ABOUT_RCT2:
-            WindowAboutSetPage(w, widgetIndex - WIDX_TAB_ABOUT_OPENRCT2);
-            break;
-    }
-}
+        auto pos = AboutWindow::GetCentrePositionForNewWindow(WW, WH);
+        auto* newWindow = WindowCreate<AboutWindow>(WC_ABOUT, pos, WW, WH, WF_RESIZABLE);
```
Comment (duncanspumpkin) [inline src/openrct2-ui/windows/About.cpp:275]: Use WindowFocusOrCreate and also use WF_CENTRE_SCREEN

File: src/openrct2-ui/windows/About.cpp [hunk 2]
```diff
@@ -73,237 +74,207 @@ static rct_widget window_about_openrct2_widgets[] = {
...
-
-    w->pressed_widgets |= (page == WINDOW_ABOUT_PAGE_RCT2) ? (1ULL << WIDX_TAB_ABOUT_RCT2) : (1ULL << WIDX_TAB_ABOUT_OPENRCT2);
-
-    WindowInitScrollWidgets(w);
-    w->Invalidate();
-}
+        return newWindow;
+    }
+    return window;
+}
```
Comment (duncanspumpkin) [inline src/openrct2-ui/windows/About.cpp:280]: Add a new line

File: src/openrct2-ui/windows/About.cpp [hunk 3]
```diff
@@ -73,237 +74,207 @@ static rct_widget window_about_openrct2_widgets[] = {
...
-        case WIDX_CLOSE:
-            window_close(w);
-            break;
-        case WIDX_TAB_ABOUT_OPENRCT2:
-        case WIDX_TAB_ABOUT_RCT2:
-            WindowAboutSetPage(w, widgetIndex - WIDX_TAB_ABOUT_OPENRCT2);
-            break;
-    }
-}
+        auto pos = AboutWindow::GetCentrePositionForNewWindow(WW, WH);
```
Comment (duncanspumpkin) [inline src/openrct2-ui/windows/About.cpp:274]: No need for the function

**Review by duncanspumpkin — APPROVED** [review]

**Pull request merged**